src/core/public/chrome/ui/header/header_breadcrumbs.tsx

```diff
@@ -40,11 +40,13 @@ import './header_breadcrumbs.scss';
 interface Props {
   appTitle$: Observable<string>;
   breadcrumbs$: Observable<ChromeBreadcrumb[]>;
+  isDarkMode: boolean | undefined;
 }
 
-export function HeaderBreadcrumbs({ appTitle$, breadcrumbs$ }: Props) {
+export function HeaderBreadcrumbs({ appTitle$, breadcrumbs$, isDarkMode }: Props) {
   const appTitle = useObservable(appTitle$, 'OpenSearch Dashboards');
   const breadcrumbs = useObservable(breadcrumbs$, []);
+  const className = !isDarkMode ? 'osdHeaderBreadcrumbs' : 'osdHeaderBreadcrumbs--dark';
```

Comment:
nit - positive conditions are a bit more readable:
```suggestion
  const className = isDarkMode ? osdHeaderBreadcrumbs--dark' : 'osdHeaderBreadcrumbs';
```